pkg/storaged/btrfs/subvolume.jsx

```diff
@@ -70,6 +194,15 @@ export function make_btrfs_subvolume_page(parent, volume, subvol) {
         });
     }
 
+    // refactor to always be mount_point_in_parent?
+    if ((mounted && !opt_ro) || (mount_point_in_parent && !mounted)) {
+        const parent_dir = mounted ? mount_point : mount_point_in_parent;
+        actions.push({
```

Comment:
I'd say the action should always be there, but disabled with an explanation if it's not possible.

Reply:
That is a good suggestion and a lot better then what we have now! That also means it becomes a secondary action in my opinion. As create will always be possible.

Reply:
I agree, but this does not seem to be possible yet with the storage dialog? 

We have one shared "action" and "disabled" only takes a promise? So I don't see how we can easily act on a value changing?
```
        function add_action(variant) {
            actions.push({
                caption: variant.Title,
                style: actions.length == 0 ? "primary" : "secondary",
                danger: def.Action.Danger || def.Action.DangerButton,
                disabled: running_promise != null || (def.Action.disable_on_error &&
                                                      errors && errors.toString() != "[object Object]"),
                clicked: progress_callback => run_action(progress_callback, variant.tag),
            });
        }
```

Reply:
So maybe we should support a `variant.disabled` as option? This feels a bit out of scope for a first version.

Reply:
Agreed this is nicer, added and added tests and found a bug!